compiler/codegen/OMRCodeGenerator.hpp

```diff
@@ -1475,9 +1475,6 @@ class OMR_EXTENSIBLE CodeGenerator
    bool getSupportsArraySet() {return _flags1.testAny(SupportsArraySet);}
    void setSupportsArraySet() {_flags1.set(SupportsArraySet);}
 
-   bool getSupportsArraySetToZero() {return _flags3.testAny(SupportsArraySetToZero);}
```

Comment:
I'm not sure we really want to get rid of this - it is a useful feature being able to allow only arrayset to zero or general arrayset. @vijaysun-omr thoughts?